apps/test/integration/levelSolutions/applab/ec_canvas.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
         'rect(80, 120, 160, 240);\n' +
         'setStrokeWidth(3);\n' +
         'setStrokeColor("red");\n' +
-        'setFillColor("yellow");\n' +
+        'setFillColor(rgb(255,0,0));\n' +
```

Comment:
The 1-2 args case will be caught when it gets passed to another function because it won't be a valid color. If users are programming in blocks, they will never get to this case because it can only have 3-4 parameters.